Src/ILGPU/Backends/OpenCL/CLCodeGenerator.Values.cs

```diff
@@ -706,5 +706,10 @@ public void GenerateCode(WarpShuffle shuffle)
         public void GenerateCode(DebugAssertOperation debug) =>
             // Invalid debug node -> should have been removed
             debug.Assert(false);
+
+        /// <summary cref="IBackendCodeGenerator.GenerateCode(LanguageEmitValue)"/>
+        public void GenerateCode(LanguageEmitValue value) =>
+            // Ignore PTX instructions.
+            value.Assert(value.LanguageKind == LanguageKind.PTX);
```

Comment:
I am not sure whether this assert is correct... 🤓

Reply:
Correct. In future, if we add a new `LanguageKind`, this code will need to be modified to either ignore the new language, or emit the correct instructions.